@ecomplus/storefront-template/template/pages/@/sections/product-block.ejs

```diff
@@ -146,6 +146,7 @@ const productJsonLd = {
     priceCurrency: _.settings.currency,
     price: _.ecomUtils.price(_.state),
     itemCondition: `http://schema.org/${(_.state.condition === 'new' ? 'New' : 'Used')}Condition`,
+    priceValidUntil: _.ecomUtils.onPromotion(_.state) && _.state.price_effective_date.end ? _.ecomUtils.formatDate(_.state.price_effective_date.end).replace(/(\d{2})\/(\d{2})\/(\d{4})/, '$3-$2-$1') : undefined ,
```

Comment:
```suggestion
    priceValidUntil: _.ecomUtils.onPromotion(_.state) && _.state.price_effective_date.end
      ? _.ecomUtils.formatDate(_.state.price_effective_date.end).replace(/(\d{2})\/(\d{2})\/(\d{4})/, '$3-$2-$1')
      : undefined,
```

💅🏾

Reply:
Além da identação, acho que você não deveria usar `_.ecomUtils.formatDate` aqui, você tá formatando pra depois desformatar, e ele vai ter o retorno variado conforme a localização do cliente (seu código só funciona se a loja for pt br).

Seria mais correto (e mais performático) usar métodos do `Date` prototype, pra não precisar mexer muito na implementação atual poderia ser partindo de https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString